### ISSUE TITLE
lr_scheduler config issue

### ISSUE THREAD

**Issue by GeonHyeock** [ashleve/lightning-hydra-template#647]
https://github.com/ashleve/lightning-hydra-template/blob/bddbc24b82ab6ccfa6243e815a49dc5bfe8d4144/src/models/mnist_module.py#L202-L213

**There is an issue with not being able to change the monitor interval frequency value of the lr_scheduler using the config.**

If I need to change the learning rate at each step, I will have to modify the module file directly.
